docs/using.md

```diff
@@ -1,166 +1,484 @@
-    level above `Phases`. Because Genny recursively traverses up the YAML to find an
-    option, most of the options can be pushed down and defined at the phase level
-    instead. The notable exceptions are `Name`, `Type`, and `Threads`,
-    which must be defined on `Actor`.
+# Further Concepts
 
-2.  `./run-genny workload -- evaluate ./src/workloads/docs/HelloWorld.yml` is your friend. `evaluate` prints out
-    the final YAML workload with all external phase definitions inlined.
 
-## Overrides
+<a id="org78b250a"></a>
 
-Genny has an override syntax for configuring workloads. When invoking Genny, you can use the `-o` option to specify an override file.
-This uses [OmegaConf](https://omegaconf.readthedocs.io/en/2.1_branch/) to merge the override file onto the workload. This functionality
-should only be used to set values that absolutely need to be specified at runtime, such as URIs for systems under test.
+## Common Actors
 
-For an example of when using overrides may be a good idea, check out our docs on [Connection Strings](./connection.md).
+There are several actors owned by TIPS which are intended for widespread use:
 
-When using overrides or any other workload preprocessing, you can always use the `evaluate` subcommand to see the final workload that will
-actually be executed.
+-   CrudActor - Used to perform CRUD operations, recording client-side metrics.
+-   RunCommand - Execute a command against the remote server. Often used for utility purposes, but metrics are collected as well.
+-   Loader - Load many documents into the remote database. Often used early in a workload to set the preconditions for testing.
+-   QuiesceActor - Quiesce a cluster, making sure common operations are complete. This is often used to reduce noise between phases.
 
+Examples with these and other actors can be found in [./src/workloads/docs](../src/workloads/docs).
 
-## Patch-Testing Genny Changes with Sys-Perf / DSI
 
-Install the [evergreen command-line client](https://evergreen.mongodb.com/settings) and put it
-in your PATH.
+<a id="org2b04b49"></a>
 
-You will create an "empty" patch of the sys-perf project and then modify that patch to use
-your custom version of genny with it.
+## AutoRun
 
-Create a patch build from the mongo repository:
 
-```sh
-cd mongo
-evergreen patch -p sys-perf
-```
+<a id="orgd0067d1"></a>
 
-You will see an output like this:
+### What is AutoRun?
 
-```
-            ID : 5c533a2732f4174bbcb8bb2e
-       Created : 35.656ms ago
-    Description : <none>
-         Build : https://evergreen.mongodb.com/patch/5c533a2732f4174bbcb8bb2e
-      Finalized : No
-```
+AutoRun is a utility to allow workload authors to determine scheduling of their workloads without having to commit to a separate repo. The utility is specifically designed for users who are using DSI through Evergreen. To use AutoRun, make sure your project has integrated DSI with Evergreen as explained [here](https://github.com/10gen/dsi/wiki/DSI-In-Evergreen).
```

Comment:
Perhaps worth noting that autorun looks in `./src/*/src/workloads` and assumes all repos including genny itself are checked out in `./src`.